js/adapt-contrib-slider.js

```diff
@@ -27,12 +27,29 @@ define([
                 this.setupModelItems();
             }
 
+            this.setupLabels();
             this.restoreUserAnswers();
             if (this.model.get('_isSubmitted')) return;
 
             this.selectItem(0, true);
         },
 
+        setupLabels: function() {
+            if (this.model.get('_labels')) return;
+
+            // Backwards compatibility
```

Comment:
I would be tempted to make the name of this function more descriptive so as to make the comment unnecessary... maybe even just `backwardsCompatibility`??

just a suggestion, but this looks a little tidier I think
```js
this.model.set('_labels', {
    _start: {
        text: this.model.get('labelStart')
    },
    _end: {
        text: this.model.get('labelEnd')
    }
});
```